src/webpack-angular-types-plugin/plugin.ts

```diff
@@ -65,26 +79,27 @@ export class WebpackAngularTypesPlugin {
         module.addDependency(codeDocDependency);
     }
 
-    // noinspection JSMethodCanBeStatic
-    private collectModulesToProcess(
-        modules: Set<Module>,
-        tsProject: Project
-    ): [string, Module][] {
-        const modulesToProcess: [string, Module][] = [];
-        for (const module of modules) {
-            const filePath = module.nameForCondition();
+    private getModuleInfoIfProcessable(
+        module: Module
+    ): ModuleInformation | null {
+        const filePath = module.nameForCondition();
 
-            // Skip null values (e.g. raw files)
-            if (!filePath) {
-                continue;
-            }
+        // Skip null values (e.g. raw files)
+        if (!filePath) {
+            return null;
+        }
 
-            // Only add modules that are part of the tsProject
-            if (tsProject.getSourceFile(filePath) && filePath.endsWith(".ts")) {
-                modulesToProcess.push([filePath, module]);
-            }
+        // Only add modules that are part of the tsProject
+        if (
+            !this.tsProject.getSourceFile(filePath) ||
+            !filePath.endsWith(".ts")
```

Comment:
Is line 95 also a candidate for adding it to line 46 as mentioned in the above comment?